src/test-data/Urls.js

```diff
@@ -1,13 +1,39 @@
 /* The following URLs are all of the URLs used in testing and in the stories. */
-export const revFourCatalog =
-  "https://raw.githubusercontent.com/usnistgov/oscal-content/master/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_catalog.json";
-export const parentUrlTestData =
-  "https://raw.githubusercontent.com/usnistgov/oscal-content/master/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_MODERATE-baseline_profile.json";
 export const backMatterTestUrl =
   "https://raw.githubusercontent.com/usnistgov/oscal-content/master/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_MODERATE-baseline_profile.json/../NIST_SP-800-53_rev4_catalog.json";
+
+export const componentDefinitionUrl =
+  "https://raw.githubusercontent.com/EasyDynamics/oscal-content/5ef5b3709045ba3aab1473813c13436e886d6185/examples/component-definition/json/example-component.json";
+
+export const defaultOscalCatalogUrl =
+  "https://raw.githubusercontent.com/usnistgov/oscal-content/master/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json";
+
+export const defaultOSCALComponentUrl =
+  "https://raw.githubusercontent.com/EasyDynamics/oscal-content/manual-fix-of-component-paths/examples/component-definition/json/example-component.json";
+
+export const defaultOSCALProfileUrl =
+  "https://raw.githubusercontent.com/usnistgov/oscal-content/master/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_MODERATE-baseline_profile.json";
```

Comment:
Is this the same as `parentUrlTestData`? Will it always be?

Reply:
Yes, this is the same as parentUrlTestData

Reply:
It seems we do not use `parentUrlTestData` anywhere. Can we just get rid of it?

Reply:
That depends on how we handle the default viewer urls

Reply:
Hey @mikeisen1 please try to let the thread authors resolve conversations. I think I still need an explanation for exactly what we plan to use this for, why the value is the same, and why it's a different variable. Are we expecting to eventually use different URLs? If the answer is "it depends" then I think we should talk about the assumptions and design and make sure we understand what it depends on and what we've chosen here. I am missing an understanding of that justification and it seems that @tuckerzp may be as well.

Reply:
I am not sure why the value is the same and they're in different variables. Both `parentUrlTestData` and `defaultOSCALProfileUrl` have been in the codebase since before I have been working on the project.

My "depends" reply was in response to Zach Tucker's reply. When I moved the default urls back to the OSCALLoader file, I kept their original scope (none of them had `export`). So, I would only delete `parentUrlTestData` if I moved `defaultOSCALProfileUrl` back to the OSCALLoader file and did not export it. As for whether they will both always be the same, I think they would only be different if we want the urls used in the stories and tests to be different from the default viewer urls.

Reply:
> Both `parentUrlTestData` and `defaultOSCALProfileUrl` have been in the codebase since before I have been working on the project.

So that is an issue that I have had to deal with a few time. A lot of changes were made and thought out long before we started on project. Using `Git Blame` and checking the history for why something was done and when is super helpful to get an idea of why something may or may not be important.

From looking purely at the code you have added, `parentUrlTestData` is not used anywhere as of now. `defaultOSCALProfileUrl` seems to have replaced all those places where it was used. Going off of conversations above, `defaultOSCALProfileUrl` is not really needed in `Urls.js`, so `parentUrlTestData` could be needed again. 

With that said, I am curious why they are the same value and in different variables. Maybe look into why they might be similar?